config/config.go

```diff
@@ -128,6 +128,8 @@ type Config struct {
 	DialRanker network.DialRanker
 
 	SwarmOpts []swarm.Option
+
+	NetworkCookie crypto.NetworkCookie
```

Comment:
I think you should rebrand this as PNET v2.
They both seems to achieve the same thing, I havn't red the code enough to know if `NetworkCookie` is private information, but that doesn't sounds very hard to ensure we never leak the cookie.